app/presenters/tree_builder_archived.rb

```diff
@@ -1,11 +1,15 @@
 module TreeBuilderArchived
   def x_get_tree_custom_kids(object, count_only, options)
-    klass  = Object.const_get(options[:leaf])
-    objects = case object[:id]
-              when "orph" then klass.all_orphaned
-              when "arch" then klass.all_archived
+    klass = Object.const_get(options[:leaf])
+    objects = if TreeBuilder.hide_vms
+                [] # hidden all VMs
+              else
+                case object[:id]
+                when "orph" then  klass.all_orphaned
+                when "arch" then  klass.all_archived
```

Comment:
I know this is previous code, but the module is named TreeBuilderArchived, but the content has both archived and orphaned.  Is that right? @kbrock Do you know?